src/SimpleList/index.js

```diff
@@ -229,23 +229,60 @@ class Row extends Component {
     return false
   }
 
+  renderImageLink(styleType) {
+    let { leftSection } = this.props
+    
+    let source = leftSection.image
+    const pointerEvents = leftSection.onPress ? "unset" : "none"
+
+    const ImageRender = (
+      <Image
+        resizeMode="cover"
+        source={source}
+        style={styleType}
+        pointerEvents={pointerEvents}
+      />
+    );
+
+    if(leftSection.onPress) {
+      return <TouchableOpacity onPress={leftSection.onPress}>{ImageRender}</TouchableOpacity>
```

Comment:
```suggestion
      return (
      <TouchableOpacity onPress={leftSection.onPress}>
          {ImageRender}
      </TouchableOpacity>
      )
```
I'd personally do something like this for readability. But this looks great otherwise!